spec/spec_helper.rb

```diff
@@ -30,6 +30,10 @@
   config.before :all do
     FactoryBot.reload
   end
+
+  config.before(:each, type: :system) do
```

Comment:
Why not use `:rack_test` for those tests that don't require javascript? It's faster.

Reply:
Because we don't specify which tests use javascript, any system could have javascript which headless chrome can handle and rack test can not.

Reply:
> we don't specify which tests use javascript

We could, it's pretty easy. And then do something like [this](https://github.com/umts/umaps-rails/blob/master/spec/rails_helper.rb#L67-L71).